ivy/array/manipulation.py

```diff
@@ -41,10 +41,48 @@ def expand_dims(
     def reshape(
         self: ivy.Array,
         shape: Union[ivy.Shape, ivy.NativeShape],
+        copy: Optional[bool] = None,
         *,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
-        return ivy.reshape(self._data, shape, out=out)
+        """ivy.Array instance method variant of ivy.reshape. This method simply wraps the
+        function, and so the docstring for ivy.reshape also applies to this method
+        with minimal changes.
+
+        Parameters
+        ----------
+        self
+            Input array to be reshaped.
+        shape
+            The new shape should be compatible with the original shape. One shape
+            dimension can be -1. In this case, the value is inferred from the length
+            of the array and remaining dimensions.
+        copy
+            boolean indicating whether or not to copy the input array. If True, the
+            function must always copy. If False, the function must never copy and must
+            raise a ValueError in case a copy would be necessary. If None, the function
+            must reuse existing memory buffer if possible and copy otherwise.
+            Default: None.
+        out
+            optional output array, for writing the result to. It must have a shape that
+            the inputs broadcast to.
+
+        Returns
+        -------
+        ret
+            Reshaped array.
+
+        Examples
+        --------
+        >>> x = ivy.array([[0., 1., 2.]])
+        >>> y = x.reshape(3, 1)
```

Comment:
on line number 78 of array/manipulation.py, the argument should be a tuple, e.g. ```x.reshape((3, 1))``` instead of ```x.reshape(3, 1)```

Reply:
Hey @Aarsh2001 . Done